README.md

```diff
@@ -149,6 +155,32 @@ subtract_omit = */.tox/*
 this will result in `*/.tox/*` not being `omit`ted (`*/.tox/*` is among the
 defaults provided by `covdefaults`).
 
+#### coverage for installed libraries
+
+By default ``covdefaults`` assumes that you want to track coverage
+in your local source tree, code that is version controlled. For libraries
+it's better idea to detect coverage on your installed version, and map that
+back to the source files. You can enable tracking installed libraries via:
+
+```ini
+[coverage:covdefaults]
+installed_libraries = tox:src virtualenv:.
```

Comment:
hmmm it's unclear to me why these are marked `tox:` / `virtualenv:` but I'll read onwards -- ideally we don't prefix these

Reply:
because you need to map them back somewhere that's potentially other than ``.``; you can also not enter it.... and it will default to ``.``

Reply:
could you explain more, I still don't understand :(

Reply:
Assume you track the coverage, you'll have files such as ``.tox/py37/lib/python3.7/site-packages/tox/__init__.py`` and then ``.tox/py38/lib/python3.8/site-packages/tox/__init__.py``. If you want to merge together these two and report the coverage you need to map back both of them to the raw file, e.g. ``src/tox/__init__.py``. You need to know where the unpackaged sources are living, or what to consider the true source file. The part after the ``:`` specifies where the installed package maps back. Coverage report will fail if you map back to a path that does not exist.

Reply:
is this used for anything besides the `paths` code that we can't have at this time?  I'd prefer something that's not stringly typed